src/views/activity/components/bridge-card/bridge-card.view.tsx

```diff
@@ -55,11 +125,27 @@ const BridgeCard: FC<BridgeCardProps> = ({
     </div>
   );
 
+  const remainingBatchesMsg: string = (() => {
+    if (
+      isAsyncTaskDataAvailable(lastVerifiedBatch) &&
+      isAsyncTaskDataAvailable(batchNumberOfL2Block)
+    ) {
+      return `Waiting for validity proof. Tx will be confirmed in ${Math.max(
+        batchNumberOfL2Block.data.sub(lastVerifiedBatch.data).toNumber(),
+        0
+      )} batches`;
+    } else if (lastVerifiedBatch.status === "failed" || batchNumberOfL2Block.status === "failed") {
+      return "Waiting for validity proof. It can take between 15 min and 1 hour";
```

Comment:
According to [this suggestion](https://0xpolygon.slack.com/archives/C04AD94JAF9/p1674486579845599?thread_ts=1674220096.230739&cid=C04AD94JAF9), this should be:

> Waiting for validity proof. This may take between 15 min and 1 hour

Reply:
Let's apply it yep

Reply:
Done